Jenkinsfile

```diff
@@ -106,6 +106,7 @@ def buildHipClangJob(Map conf, compiler){
                         mkdir build
                         rm -rf install
                         mkdir install
+                        rm -rf src/kernels/*.ufdb.txt
```

Comment:
pleas add src/kernels/miopen_1.0.0.udb

Reply:
Do you really see the user-perf-db leftovers???

Reply:
Locally yes

Reply:
Correct.